api/v1beta1/machinedeployment_types.go

```diff
@@ -211,6 +216,31 @@ type MachineRollingUpdateDeployment struct {
 
 // ANCHOR_END: MachineRollingUpdateDeployment
 
+// ANCHOR: RemediationStrategy
+
+// RemediationStrategy allows to define how the MachineSet can control scaling operations.
+type RemediationStrategy struct {
+	// MaxInFlight determines how many in flight remediations the MachineSet should issue at
+	// any given point in time.
+	//
+	// This field gives control over how many Machines should be remediated (deleted),
+	// at any given point in time during the MachineSet's reconciliation.
+	//
+	// Remediation only happens on the MachineSet with the current revision, during rollout
+	// operations, older MachineSets aren't allowed to remediate while a rollout is in progress.
+	// Unhealthy machines are always prioritized during scale down operations over healthy ones.
```

Comment:
I thought this entire field is about remediations, then here we talk about scale downs? Does this field affect scale downs outside of remediations?

Reply:
This was a suggestion by @fabriziopandini, should we remove it?

Reply:
So the main point here is not that this field affects that, but just to clarify that this is how it works in general?
Then I would propose:
```
Note: In general (independent of remediations), unhealthy machines are always prioritized during scale down operations over healthy ones.
```

Reply:
done